CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [Release 2.23.0](https://github.com/aws-amplify/aws-sdk-android/releases/tag/release_v2.23.0)
+
+### Features
+- **aws-android-sdk-core:** update models to latest (#2445)
```

Comment:
Hmm, looks like the minor version got bumped because of this change.  This is not actually a new feature though - we really want the version to be `2.22.7`.

Reply:
The version is automated bumped by the bot. Is the major version bumped by default?